libdnf/repo/repo_downloader.cpp

```diff
@@ -526,11 +326,11 @@ void RepoDownloader::load_local() try {
 
 /// Returns a librepo handle, set as per the repo options.
 /// Note that destdir is None, and the handle is cached.
-LrHandle * RepoDownloader::get_cached_handle() {
+LibrepoHandle * RepoDownloader::get_cached_handle() {
```

Comment:
Would returning a reference here be a bit nicer? At least the caller would know they're not getting a `nullptr`.

Reply:
OK